mas_perception_libs/ros/include/mas_perception_libs/point_cloud_utils.h

```diff
@@ -0,0 +1,53 @@
+/*!
+ * @copyright 2018 Bonn-Rhein-Sieg University
+ *
+ * @author Minh Nguyen
+ *
+ */
+#ifndef MAS_PERCEPTION_LIBS_POINT_CLOUD_UTILS_H
+#define MAS_PERCEPTION_LIBS_POINT_CLOUD_UTILS_H
+
+#include <opencv/cv.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <mas_perception_libs/aliases.h>
+#include <mas_perception_libs/bounding_box_2d.h>
+
+namespace mas_perception_libs
+{
+/*!
+ * @brief extract a CV image from a sensor_msgs/PointCloud2 object
+ */
+cv::Mat
+cloudMsgToCvImage(const sensor_msgs::PointCloud2 &pCloudMsg);
+
+/*!
+ * @brief extract a CV matrix containing (x, y, z) coordinates from a pcl::PointCloud object within a region
+ *        defined by a BoundingBox2D object
+ */
+cv::Mat
+cropCloudToXYZ(const PointCloud &pCloud, BoundingBox2D &pBox);
```

Comment:
all these BoundingBox2D objects should be const ref too I think

Reply:
similar reason to above